cirq/ops/pauli_string.py

```diff
@@ -115,6 +130,24 @@ def __rmul__(self, other):
             return PauliString(self._qubit_pauli_map, self._coefficient * other)
         return NotImplemented
 
+    def __add__(self, other):
+        from cirq.ops.linear_combinations import PauliSum
```

Comment:
This method should basically just be delegating to `PauliSum`. Just `return PauliSum.from(self).__add__(other)`.

Reply:
done